lfr/7pin_cardinal_demo/runjob_sawtooth.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#PBS -N HCFwork
```

Comment:
change this to something like `<name of your project>`

Reply:
done